http/server.go

```diff
@@ -216,18 +216,19 @@ func (s *Server) acceptQuoteHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	p := getProviderByAddress(s.providers, quote.LPRSKAddr)
-	signB, err := p.SignHash(hashBytes)
+	cfuCost, err := s.rsk.EstimateGas(quote.ContractAddr, quote.Value, []byte(quote.Data))
```

Comment:
This should be (quote.gasLimit + CFUExtraGas) * gasPrice + q.value

Reply:
fixed.